app/models/requests/requestable.rb

```diff
@@ -69,6 +69,10 @@ def missing?
       return true if item[:status] == 'Missing'
     end
 
+    def lewis?
```

Comment:
scinb is the "New Book Shelf". There are only 8 items there. I think we want to leave this one out of the pageable locations.